src/oasis-nexus/api.ts

```diff
@@ -755,7 +752,7 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
                               event.body.amount.Amount,
                               paraTimesConfig[runtime].decimals,
                             ),
-                            Denomination: getTickerForNetwork(network),
+                            Denomination: getTokensForScope({ network, layer: runtime })[0].ticker, // TODO find this out from event data
```

Comment:
We need to differentiate decimals based on denomination. If EUROe has the same number of decimals as TEST, that is only coincidentally and unreliable

Reply:
Yes, decimals are not yet solved. However this code has been updated later in #1279